source/conf.py

```diff
@@ -45,8 +45,8 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'OSL Wiki'
-copyright = u'2014, Oregon State University'
+project = 'OSL Wiki'
+copyright = '2014, Oregon State University'
```

Comment:
If you're doing this, you might as well update this to the following:
```suggestion
copyright = '2014-2019, Oregon State University'
```